README.md

```diff
@@ -25,33 +24,81 @@ for your environment:
+commands, you will select paths by specifying a range (in the format of `START-END`).  Ranges
+can start at index 0 and their start/end points are inclusive.  Thefore, a range `4-6` would
+selct 3 paths. Using the command below, you can find the max endpoint for ranges in a paths 
+file.
+
+    $ wc -l paths/2015-18.wat.paths 
+    38609 paths/2015-18.wat.paths
+
+The 2015-18 paths file has 38609 different paths.  A range of `0-38608` would select all 
+paths in the file.
+
 ### Copy CommonCrawl data from AWS into HDFS
 
-CommonCrawl data sets are hosted in S3.  The following command will run a Spark job that copies
-data files from AWS into HDFS `init/` and `load/` directoires.  This script is configured by 
-`conf/data.yml` where you can specify the number of files to copy into each directory:
+After you have retrieved a paths file, you can run the command below to run a Spark job that copies 
+data files from S3 to a directory in your local HDFS instance.  The command below will copy 3 files
+in the file range of `4-6` of the `2015-18` paths file into the HDFS directory `/cc/data/a`.  
+CommonCrawl data files are large (~330 MB each) so be careful not copy too many if you are running 
+in a development environment.
 
-    ./bin/webindex copy
+    ./bin/webindex copy 2015-18 4-6 /cc/data/a
 
-### Initialize Fluo & Accumulo with data
+If you need to create multiple data sets, you can run this command with different ranges to create 
+directories with different data:
 
-After you have copied data into HDFS, run the following command to run a Spark job that will initialize 
-Fluo and Accumulo with data in your HDFS `init/` directory.
+    ./bin/webindex copy 2015-18 7-8 /cc/data/b
+
+### Initialize Fluo & Accumulo
+
+After you have copied data into HDFS, run the following command to initialize everything and 
+start Fluo.
 
     ./bin/webindex init
 
+Optionally, you can add a HDFS directory (that contains data) to the end of the command, the `init` 
+command will also run a Spark job that will initialize Fluo and Accumulo with this data.
+    
+    ./bin/webindex init /cc/data/a
+
 ### Load files into Fluo
 
-The `init` command can only be run on an empty cluster.  To add more data, run the `load` which starts
-a Spark job that loads Fluo with data in your HDFS `load/` directory.  Fluo will incrementally process 
-this data and update Accumulo.
+The `init` command should only be run on an empty cluster.  To add more data, you can run the 
+`loaddir` or `loadurls` commands.  Both start a Spark job that parses CommonCrawl and inserts 
+this data into Fluo.  Fluo will incrementally process this data and export indexes to Accumulo.
+
+The `loaddir` command below loads data stored in the HDFS directory `/cc/data/b` into Fluo:
+
+    ./bin/webindex loaddir /cc/data/b
+
+The `loadurls` command below downloads the URLs hosted in S3 in the `9-10` range of the `2015-18` 
```

Comment:
how about i call them `load-hdfs` and `load-s3`?  I want to keep the commands in lower case.  I should probably also highlight that we are loading from HDFS and a not a local dir.